apps/motionscope/analyze.py

```diff
@@ -128,7 +128,12 @@ def func(val):
     def export_gs(self, filename):
         data = self.data_frame()
         gtc = self.gcloud.get_interface("KtabularClient")
-        sheet = gtc.create(filename,data)
+        sheet = gtc.lookup(filename)
+        if (sheet != None):
+            gtc.clear(sheet)
+            gtc.append_data(sheet, data)
+        else:
+            sheet = gtc.create(filename,data)
         return gtc.get_url(sheet)
 
     def data_frame(self):
```

Comment:
Please check line 46 of analyze.py
        self.gcloud = Gcloud("/home/pi/vizy/etc")
should be:
       self.gcloud = Gcloud(self.kapp.etcdir)